02-wrangling.Rmd

```diff
@@ -304,6 +317,76 @@ From this visualization, we can see that in Calgary, Edmonton, Toronto and Vanco
 
 Is there only one shape for tidy data for a given data set? Not necessarily! It depends on the statistical question you are asking and what the variables are for that question. For tidy data, each variable should be its own column. So, just as it's essential to match your statistical question with the appropriate data analysis tool (classification, clustering, hypothesis testing, etc.). It's important to match your statistical question with the appropriate variables and ensure they are represented as individual columns to make the data tidy.
 
+
+
+## More ways to create subsets of data frames with `select` & `filter`
```

Comment:
I think we need a snappier header - it is difficult to read the whole header in the book table of contents on the left-handside when viewed as an html book.